io.openems.edge.goodwe.et/src/io/openems/edge/goodwe/et/ess/GoodWeEtBatteryInverterImpl.java

```diff
@@ -405,12 +405,12 @@ public void applyPower(int activePower, int reactivePower) throws OpenemsNamedEx
 			activePower = 0;
 		} else {
 			if (activePower < 0) {
-				// ActivePower is negative or zero -> CHARGE
+				// ActivePower is negative -> CHARGE
```

Comment:
@venu-sagar: You have inverted the logic here. The handling for `activePower == 0` is now the same as discharge mode. is that ok/what you want?